analytics/test_api.py

```diff
@@ -0,0 +1,39 @@
+# test_analytics_api.py
+import unittest
+from flask import json
+from app import app
+
+class AnalyticsAPITestCase(unittest.TestCase):
+    def setUp(self):
+        self.app = app.test_client()
```

Comment:
I'm not sure this sets up the db mock properly - try implementing the way we caught up about!